config/author-config-old.csv

```diff
@@ -0,0 +1,9 @@
+Repository's Location,Branch,Author's GitHub ID,Author's Emails,Author's Display Name,Author's Git Author Name,Ignore Glob List
```

Comment:
I think this file can be removed altogether

Reply:
Ok, will update after I'm back from reservists.